README.md

```diff
@@ -72,20 +72,44 @@ It's also possible to build and use a locally built image. See the
 
 ## Securing the Docker ACAP using TLS
 
-The Docker ACAP can be run either unsecured or in TLS mode. The Docker ACAP uses
-TLS as default. Use the "Use TLS" dropdown in the web interface to switch
-between the two different modes. It's also possible to toggle this option by
-calling the parameter management API in [VAPIX](https://www.axis.com/vapix-library/) and setting the
-`root.dockerdwrapper.UseTLS` parameter to `yes` or `no`. The following commands would enable TLS:
+The Docker Compose ACAP application can be run in either TLS mode or unsecured mode. The Docker Compose
```

Comment:
We should still inform about that not both TCP and IPC socket can be set to "no"
It's hard to find a good spot for it in this version of the README but perhaps at the end of the "Using the Docker ACAP" section.
Something like: The application can provide a TCP socket if the TCP Socket setting is set to "yes" and an IPC socket if the IPC Socket setting is set to "yes". Please be aware that at least one socket must be selected for the application to start.

Reply:
I thought it would be good to keep the similar information  in TLS section as well since we have "Using the Docker ACAP"  after TLS part. So i have updated the both sections.